sdk/communication/communication-phone-numbers/recordings/browsers/phonenumbersclient__lro__purchase_and_release_aad/recording_can_purchase_and_release_a_phone_number.json

```diff
@@ -4,486 +4,264 @@
    },
    "requestBody": null,
    "status": 200,
-   "response": "{\"operationType\":\"purchase\",\"status\":\"running\",\"resourceLocation\":null,\"createdDateTime\":\"2022-03-03T00:10:53.265642+00:00\",\"id\":\"purchase_sanitized\",\"lastActionDateTime\":\"0001-01-01T00:00:00+00:00\"}",
+   "response": "{\"operationType\":\"purchase\",\"status\":\"running\",\"resourceLocation\":null,\"createdDateTime\":\"2022-05-12T16:10:42.2643332+00:00\",\"id\":\"purchase_sanitized\",\"lastActionDateTime\":\"0001-01-01T00:00:00+00:00\"}",
    "responseHeaders": {
-    "api-supported-versions": "2021-03-07, 2022-01-11-preview2",
+    "api-supported-versions": "2021-03-07, 2022-01-11-preview2, 2022-06-01-preview",
     "content-type": "application/json; charset=utf-8",
-    "date": "Thu, 03 Mar 2022 00:11:03 GMT",
-    "ms-cv": "7hwqZ5cYukuHHWIFL3AkrQ.0",
-    "request-context": "appId=",
+    "date": "Thu, 12 May 2022 16:10:54 GMT",
+    "ms-cv": "UjrFaSNUpECnAS3/sRpsEw.0",
     "strict-transport-security": "max-age=2592000",
-    "x-azure-ref": "0lwcgYgAAAAAMxGW56nCdSrMhnhczAVB9TEFYMzExMDAwMTA4MDQ3ADlmYzdiNTE5LWE4Y2MtNGY4OS05MzVlLWM5MTQ4YWUwOWU4MQ==",
+    "x-azure-ref": "0jjF9YgAAAABvBzO5XLcqT4SCZNVB0pIRUFJHMDFFREdFMDcyMQBjYzkyNzU4ZC0wNWY3LTRhZDYtYWE1ZS0wZmE5NzE4ZDg5ODU=",
     "x-cache": "CONFIG_NOCACHE",
-    "x-processing-time": "464ms"
+    "x-processing-time": "437ms"
    }
   },
   {
    "method": "GET",
    "url": "https://endpoint/phoneNumbers/operations/purchase_sanitized",
    "query": {
-    "api-version": "2022-01-11-preview2"
+    "api-version": "2022-06-01-preview"
    },
    "requestBody": null,
    "status": 200,
-   "response": "{\"operationType\":\"purchase\",\"status\":\"notStarted\",\"resourceLocation\":null,\"createdDateTime\":\"2022-03-03T00:10:53.265642+00:00\",\"id\":\"purchase_sanitized\",\"lastActionDateTime\":\"0001-01-01T00:00:00+00:00\"}",
+   "response": "{\"operationType\":\"purchase\",\"status\":\"succeeded\",\"resourceLocation\":null,\"createdDateTime\":\"2022-05-12T16:10:42.2643332+00:00\",\"id\":\"purchase_sanitized\",\"lastActionDateTime\":\"0001-01-01T00:00:00+00:00\"}",
    "responseHeaders": {
-    "api-supported-versions": "2021-03-07, 2022-01-11-preview2",
+    "api-supported-versions": "2021-03-07, 2022-01-11-preview2, 2022-06-01-preview",
     "content-type": "application/json; charset=utf-8",
-    "date": "Thu, 03 Mar 2022 00:11:05 GMT",
-    "ms-cv": "v9DQ5NJpRUaiM6EdjGCvcg.0",
-    "request-context": "appId=",
+    "date": "Thu, 12 May 2022 16:10:57 GMT",
+    "ms-cv": "3kReAgSp5ESug/MQYgD1OQ.0",
     "strict-transport-security": "max-age=2592000",
-    "x-azure-ref": "0mQcgYgAAAADLoyqXw0q2RZ7C4/hVr8TgTEFYMzExMDAwMTA4MDQ3ADlmYzdiNTE5LWE4Y2MtNGY4OS05MzVlLWM5MTQ4YWUwOWU4MQ==",
+    "x-azure-ref": "0kDF9YgAAAAAnVtBjR1VpQYJa09ats8JlUFJHMDFFREdFMDcyMQBjYzkyNzU4ZC0wNWY3LTRhZDYtYWE1ZS0wZmE5NzE4ZDg5ODU=",
     "x-cache": "CONFIG_NOCACHE",
-    "x-processing-time": "382ms"
-   }
-  },
-  {
-   "method": "GET",
```

Comment:
This is because of polling. The previous recording got the response after a long time, probably because the processing took a lot of time. The new request succeeded quickly, so we don't see the polling requests with status "notStarted".